pkg/operator/baremetal_config.go

```diff
@@ -76,6 +85,35 @@ func getBaremetalProvisioningConfig(dc dynamic.Interface, configName string) (Ba
 		glog.Errorf("provisioningOSDownloadURL not found in Baremetal provisioning CR %s", configName)
 		return BaremetalProvisioningConfig{}, err
 	}
+	// If provisioningNetwork is not provided, set it to a default of "managed".
+	provisioningNetwork, foundNetwork, err := unstructured.NestedString(provisioningSpec, "provisioningNetwork")
+	if !foundNetwork {
+		glog.Infof("provisioningNetwork not found in Baremetal provisioning CR %s", configName)
+		provisioningNetwork = provisioningNetworkManaged
+	}
+	if err != nil {
+		glog.Errorf("provisioningNetwork not found in Baremetal provisioning CR %s", configName)
+		return BaremetalProvisioningConfig{}, err
+	}
+	provisioningNetworkCIDR, foundCIDR, err := unstructured.NestedString(provisioningSpec, "provisioningNetworkCIDR")
+	if !foundCIDR {
+		glog.Infof("provisioningNetworkCIDR not found in Baremetal provisioning CR %s", configName)
+		provisioningNetworkCIDR = ""
```

Comment:
Can we base the validation off the value of provisioningNetwork? If it's anything but disabled we should get a valid value.

Reply:
Done.